### ISSUE TITLE
Broncholith

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6689]
**Preferred term label:**
Broncholith

**Synonyms**

**Definition (free text, please give PubMed ID)**
A broncholith, a calcified peribronchial lymph node that erodes into an adjacent bronchus, is most often the consequence of Histoplasma or tuberculous infection. The imaging appearance is of a small calcific focus in or immediately adjacent to an
airway, most frequently the right middle lobe bronchus. Broncholiths are readily identified on CT scans. Distal obstructive changes may include atelectasis, mucoid impaction, and bronchiectasis

PMID:18195376

Comment
See Figure 15 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983
Abnormal bronchus morphology HP:0025426

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
